toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -72,7 +74,11 @@ TFilePath getGlobalRoot() {
                          ".app/Contents/Resources/configfarmroot.txt";
 #else
   // set path to something suitable for most linux (Unix?) systems
+#ifdef FREEBSD
+  std::string unixpath = "/usr/local/etc/" + tver.getAppName() + "/opentoonz.conf";
+#else
   std::string unixpath = "/etc/" + tver.getAppName() + "/opentoonz.conf";
+#endif
```

Comment:
Isn't it better to make this a configurable value in CMake? That way someone building this can change it to whatever they want.

The idea being that each OS would also have default values, such as `/usr/local/etc` for FreeBSD.

Reply:
Better leave it as is and make all other changes in new PR, this PR is too big and take to many time.